integration-tests/bin/load-integration-data.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+## Use the Environment var or the default
+if [[ -z "${ELASTIC_SEARCH_HOST}" ]]; then
+    ES_HOST="http://localhost:9200"
+else
+    ES_HOST=${ELASTIC_SEARCH_HOST}
+fi
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+## Wait until docker is up.
+echo "Waiting for ES Service at ${ES_HOST} to Start"
+until $(curl --output /dev/null --silent --head --fail "${ES_HOST}"); do
+    printf '.'
+    sleep 1
+done
+echo "ES Service is up"
+
+# First wait for ES to start...
+response=$(curl --write-out %{http_code} --silent --output /dev/null "${ES_HOST}")
+
+until [ "$response" = "200" ]; do
+    response=$(curl --write-out %{http_code} --silent --output /dev/null "${ES_HOST}")
+    >&2 echo "Elastic Search is unavailable - sleeping"
+    sleep 1
+done
+
+# next wait for ES status to turn to Green
+health_check="curl -fsSL ${ES_HOST}/_cat/health?h=status"
+health=$(eval $health_check)
+echo "Waiting for ES status to be ready"
+until [[ "$health" = 'green' ]]; do
+    >&2 echo "Elastic Search is not ready - sleeping"
+    sleep 10
+    health=$(eval $health_check)
+done
+echo "ES status is $health"
+
+echo "Loading the index mappings"
+pushd $DIR/../data/source >/dev/null
+HDR="Content-Type: application/json"
+response=$(curl -s -H "$HDR" -XPUT ${ES_HOST}/cgov -d @cgov-mapping.json)
+if [[ ! $response =~ acknowledged[^a-zA-Z]+true ]]; then
+    echo Failure loading search mappings
+    echo $response
+fi
+response=$(curl -s -H "$HDR" -XPUT ${ES_HOST}/autosg -d @autosg-mapping.json)
+if [[ ! $response =~ acknowledged[^a-zA-Z]+true ]]; then
+    echo Failure loading autosuggest mappings
+    echo $response
+fi
+
+echo "Loading the records"
```

Comment:
Please follow the pattern of building the command into a variable and then capture the output of evaluating it in another variable.

The `load-integration-data.sh` script will be run in GitHub actions and we need to be able to see the output.  

```
MAPPING_LOAD_CMD="curl -s -H \"Content-Type: application/x-ndjson\" -XPUT ${ES_HOST}/autosg -d @autosg-mapping.json"
mapping_output=$(eval MAPPING_LOAD_CMD)
echo $mapping_output
```

Please use the [trial listing api's load-integratation-data.sh script](https://github.com/NCIOCPL/clinical-trials-listing-api/blob/develop/integration-tests/bin/load-integration-data.sh#L40-L48) as a reference.

It's already a known working example that works in all known environments.

Reply:
OK, I'll back out my approach. I was trying to follow up on the `## TODO: Actually check for errors.` but I can do it the way that example did it.

Reply:
The script has been rewritten to match the reference script.